test/plain.jl

```diff
@@ -149,6 +151,8 @@ ucoder = read(fr, "ucode")
 @assert ucode == ucoder
 salut_splitr = read(fr, "salut_split")
 @assert salut_splitr == salut_split
+salut_splitr = read(fr, "salut_2d")
+@assert salut_2d == salut_2d
```

Comment:
no, you're testing `a == a` here, the lhs and rhs should be different variables for this to mean anything (other than maybe a nan check)